core/transaction.go

```diff
@@ -229,49 +229,33 @@ func (l *L1HandlerTransaction) signature() []*felt.Felt {
 func transactionHash(transaction Transaction, n utils.Network) (*felt.Felt, error) {
 	switch t := transaction.(type) {
 	case *DeclareTransaction:
-		return declareTransactionHash(t, network)
+		return declareTransactionHash(t, n)
```

Comment:
I think these belong to a previous commit?

https://github.com/NethermindEth/juno/pull/560/commits/2387f7eedaf9c5787c177977c04723f65fa7e651#diff-65ddbfca57544ad3aaedea5dbb6e4911bbe40753fb680c41a527dccff59311ebL229

Reply:
done